rest/route/agent.go

```diff
@@ -1328,49 +1328,3 @@ func (h *setDownstreamParamsHandler) Run(ctx context.Context) gimlet.Responder {
 
 	return gimlet.NewJSONResponse(fmt.Sprintf("Downstream patches for %v have successfully been set", p.Id))
 }
-
-// POST /task/{task_id}/data/{name}
-type insertTaskJsonHandler struct {
-	taskID  string
-	name    string
-	rawData map[string]interface{}
-}
-
-func makeInsertTaskJSON() gimlet.RouteHandler {
-	return &insertTaskJsonHandler{}
-}
-
-func (h *insertTaskJsonHandler) Factory() gimlet.RouteHandler {
-	return &insertTaskJsonHandler{}
-}
-
-func (h *insertTaskJsonHandler) Parse(ctx context.Context, r *http.Request) error {
-	if h.taskID = gimlet.GetVars(r)["task_id"]; h.taskID == "" {
-		return errors.New("missing task ID")
-	}
-	h.name = gimlet.GetVars(r)["name"]
-
-	if err := utility.ReadJSON(r.Body, &h.rawData); err != nil {
-		return errors.Wrapf(err, "reading raw data from request")
-	}
-	return nil
-}
-
-func (h *insertTaskJsonHandler) Run(ctx context.Context) gimlet.Responder {
-	t, err := task.FindOneId(h.taskID)
-	if err != nil {
-		return gimlet.MakeJSONInternalErrorResponder(errors.Wrapf(err, "finding task '%s'", h.taskID))
-	}
-	if t == nil {
-		return gimlet.MakeJSONErrorResponder(gimlet.ErrorResponse{
-			StatusCode: http.StatusNotFound,
-			Message:    fmt.Sprintf("task '%s' not found", h.taskID),
-		})
-	}
-
-	if err = model.InsertTaskJSON(t, h.name, h.rawData); err != nil {
```

Comment:
I think we can get rid of this function and also maybe this collection (in which case we can drop the collection after it's deployed)

Reply:
done